.eslintrc.yml

```diff
@@ -0,0 +1,15 @@
+extends:
+- standard
+
+root: true
+
+env:
+  mocha: true
+
+rules:
```

Comment:
Let's stick to standardjs - I don't mind removing the commas or the extra space before functions's parens.

Reply:
Alright, I'll fix that. Just wanted not to change your coding style, which is why I added those two exceptions.